src/modules/rlm_mschap/rlm_mschap.c

```diff
@@ -1483,10 +1483,10 @@ static rlm_rcode_t mschap_error(rlm_mschap_t *inst, REQUEST *request, unsigned c
  *	find_nt_password() - try and find a correct NT-Password
  *	attribute, or calculate one if possible.
  */
-static bool CC_HINT(nonnull (1, 2, 4)) find_nt_password(rlm_mschap_t *inst,
-							REQUEST *request,
-							VALUE_PAIR *password,
-							VALUE_PAIR **ntpw)
+static inline bool CC_HINT(nonnull (1, 2, 4)) find_nt_password(rlm_mschap_t *inst,
```

Comment:
inline shouldn't be used.  C compilers are generally smart enough to do the right thing.